### PR TITLE
Add functionality to zoom a profile picture

### PR DESCRIPTION
**Pull request by niteshbalusu11**
Signed-off-by: Nitesh Balusu <niteshbalusu@icloud.com>

Added ability to zoom a profile picture

### CODE REVIEW

**Comment by niteshbalusu11** [comment]
https://user-images.githubusercontent.com/84944042/209376207-54b6c20e-2e0a-46eb-9b8b-276d13adda9b.mov

**Comment by jb55** [comment]
perfect!

File: damus/Views/ProfilePicView.swift [hunk 1]
```diff
@@ -37,43 +37,78 @@ struct ProfilePicView: View {
...
             let url = URL(string: pic)
-            
-            KFAnimatedImage(url)
-                .configure { view in
-                    view.framePreloadCount = 1
-                }
-                .placeholder { _ in
-                    Placeholder
+
+            if let tag = self.imageTag {
```
Comment (jb55) [inline damus/Views/ProfilePicView.swift:62]: what's the purpose of this tag?

**Review by jb55 — COMMENTED** [review]
needs rewrite since all this got changed :(

**Comment by niteshbalusu11** [comment]
> needs rewrite since all this got changed :(

I submitted the PR to the profile-edit branch. Wrong branch? Doesn't show any merge conflicts.

**Comment by niteshbalusu11** [comment]
Hmm looks like profile editing for merged?

**Comment by jb55** [comment]
please target master for merges

**Comment by niteshbalusu11** [comment]
I will resubmit this.

**Pull request closed without merging**